nautobot/dcim/views.py

```diff
@@ -3206,3 +3209,25 @@ class PowerFeedBulkDeleteView(generic.BulkDeleteView):
     queryset = PowerFeed.objects.prefetch_related("power_panel", "rack")
     filterset = filters.PowerFeedFilterSet
     table = tables.PowerFeedTable
+
+
+class RedundancyGroupUIViewSet(NautobotUIViewSet):
+    model = RedundancyGroup
+    # bulk_create_form_class = forms.*CSVForm
+    # bulk_update_form_class = forms.*BulkEditForm
+    filterset_class = filters.RedundancyGroupFilterSet
+    filterset_form_class = forms.RedundancyGroupFilterForm
+    form_class = forms.RedundancyGroupForm
+    queryset = RedundancyGroup.objects.prefetch_related("status").annotate(
+        member_count=count_related(Device, "redundancy_group")
+    )
+    serializer_class = serializers.RedundancyGroupSerializer
+    table_class = tables.RedundancyGroupTable
+    lookup_field = "pk"
+
+    def get_extra_context(self, request, instance):
+        members = Device.objects.restrict(request.user).filter(redundancy_group=instance)
```

Comment:
Please include call to `context = super.get_extra_context()` here and add `members` to that.

Reply:
🦤 👍